runtime/compiler/control/CompilationThread.cpp

```diff
@@ -11595,7 +11595,7 @@ TR::CompilationInfo::storeAOTInSharedCache(
 #ifdef COMPRESS_AOT_DATA
       try 
          {
-         if (!comp->getOption(TR_DisableAOTBytesCompression))
+         if (!comp->getOption(TR_DisableAOTBytesCompression) && !comp->isRemoteCompilation())
```

Comment:
Another way of doing this is to set `TR_DisableAOTBytesCompression` when options are processed in JITaaS mode. Look for `J9::Options::setupJITaaSOptions()` in `J9Options.cpp`

Reply:
With fa2aacc3706a002f2ba096ff6694238597b90180 we will now be able to run compression code in AOT. So this is no longer needed.